File: exams/u2/ona/Exam/src/main/java/ec/edu/espe/exam/model/Banks.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.exam.model;

import javax.swing.text.Document;
import javax.xml.crypto.Data;

/**
 *
 * @author leydi
 */
public class Banks {
    private int id;
    private String name;
    private String lastName;
    private int amount;

    public Banks() {
    }

    
    public Banks(int id, String name, String surnam, Data data) {
        this.id = id;
        this.name = name;
        this.lastName = lastName;
        this.amount = amount;
    } 
    

    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the name
     */
    public String getName() {
        return name;
    }

    /**
     * @param name the name to set
     */
    public void setName(String name) {
        this.name = name;
    }

    public String getLastName() {
        return lastName;
    }

    
    public void setLastName(String lastName) {    
        this.lastName = lastName;
    }

    public int getAmount() {
        return amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }
    
}

File: ws/ona/u3/ws35-36/Banks.py
# banks.py
class Banks:
    def __init__(self, id=None, name=None, last_name=None, amount=None):
        self._id = id
        self._name = name
        self._last_name = last_name
        self._amount = amount
    
    @property
    def id(self):
        return self._id
    
    @id.setter
    def id(self, value):
        self._id = value

    @property
    def name(self):
        return self._name
    
    @name.setter
    def name(self, value):
        self._name = value

    @property
    def last_name(self):
        return self._last_name
    
    @last_name.setter
    def last_name(self, value):
        self._last_name = value

    @property
    def amount(self):
        return self._amount
    
    @amount.setter
    def amount(self, value):
        self._amount = value
